Mage/src/main/java/mage/cards/repository/CardInfo.java

```diff
@@ -149,6 +151,7 @@ public CardInfo(Card card) {
 
         this.doubleFaced = card.isTransformable() && card.getSecondCardFace() != null;
         this.nightCard = card.isNightCard();
+        this.meldCard = card.isMeldCard();
```

Comment:
Looks like no needs in `Card::meldCard` field. You can use here `card instanceOf MeldCard` like other code does:

![shot_240407_182319](https://github.com/magefree/mage/assets/8344157/7a35dda8-c875-43c6-917d-8e71881978f0)

It's a standard code style:
![shot_240407_182455](https://github.com/magefree/mage/assets/8344157/a87dd766-2d60-4bb2-8c4c-f4caf6682aef)